packages/mongodb-chatbot-server/src/processors/makeDefaultReferenceLinks.test.ts

```diff
@@ -59,7 +59,6 @@ describe("makeDefaultReferenceLinks()", () => {
         title: "https://mongodb.com/docs/realm/sdk/node/",
         url: "https://mongodb.com/docs/realm/sdk/node/",
         metadata: {
-          sourceName: "realm",
```

Comment:
These tests are failing in CI because the `sourceName` value is still present

Reply:
think some gitfunk. will 2x check and revert